lib/api/LogManagerImpl.cpp

```diff
@@ -623,7 +623,6 @@ namespace MAT_NS_BEGIN
         LOG_INFO("SetContext");
         EventProperty prop(value, piiKind);
         m_context.SetCustomField(name, prop);
-        m_context.SetCustomField(name, prop);
```

Comment:
This was also a glitch.. Probably someone merged the same line twice. Fixing it here.